app/src/main/java/edu/rit/csh/agargiulo/Gatekeeper/HttpsConnector.java

```diff
@@ -59,6 +60,7 @@ private URI createUrl (String path)
 		try
 		{
 			postUri = URIUtils.createURI("https", POST_HOST, 0, path, null, null);
+            Log.wtf("CSH-Gatekeeper", POST_HOST);
```

Comment:
Oh it's not necessary at all, it's from debugging to make sure the endpoint was changed, I thought I removed it already. Good catch.